include/openbmc_dbus_rest.hpp

```diff
@@ -2215,48 +2216,6 @@ template <typename... Middlewares> void requestRoutes(Crow<Middlewares...> &app)
                 handleDBusUrl(req, res, objectPath);
             });
 
-    BMCWEB_ROUTE(app, "/xyz/openbmc_project/user/root/action/SetPassword")
```

Comment:
Doesn't xcat need this?

Reply:
Yes, it does.  This is dead code, meaning there is no way for control to flow into it.  I explained in the commit message that this API is handled by a special case in the route handler for /xyz/<path>.  That was added in base OP940 and remains enabled with this set of changes (with an accompanying the authority fix in this PR).

Reply:
> This is dead code, meaning there is no way for control to flow into it

this API is handled by a special case in the route handler for /xyz/, Do you mean to say that we don't need this?
Why? I am missing something.

Reply:
@joseph-reynolds I got it, It seems we don't need this.

Reply:
It would have been good if we have different commit for reverting some PR.